lib/billing_facade_client.rb

```diff
@@ -0,0 +1,62 @@
+require 'faraday'
+require 'bigdecimal'
+
+module BillingFacadeClient
+
+  def self.send_event(work_order, name)
+    connection.post("/events", {eventName: name, workOrderId: work_order.id}.to_json)
+    return true if r.status==200
+    return false
+  end
+
+  def self.validate_single_value(path)
+    r = connection.get(path)
+    return false unless r.status == 200
+    response = JSON.parse(r.body).symbolize_keys
```

Comment:
Just FYI, JSON.parse does have an option to symbolize the keys for you:

`JSON.parse(r.body, symbolize_names: true)`

Reply:
It took me a while to realise it was symbolize_names, instead of symbolize_keys 👍